### PR TITLE
SDK-1283. Crash in MegaClient::getRecentActions using Android app

### PR DESCRIPTION
**Pull request by jgandres**

### CODE REVIEW

**Comment by jgandres** [comment]
trigger compilation

File: src/posix/fs.cpp [hunk 1]
```diff
@@ -1343,15 +1343,13 @@ bool PosixFileSystemAccess::chdirlocal(LocalPath& name) const
 }
 
 // return lowercased ASCII file extension, including the . separator
-bool PosixFileSystemAccess::getextension(const LocalPath& filename, char* extension, size_t size) const
```
Comment (polmr) [inline src/posix/fs.cpp:1346]: same change should be carried out in Windows fs.cpp

File: include/mega/win32/megafs.h [hunk 2]
```diff
@@ -78,7 +78,7 @@ class MEGA_API WinFileSystemAccess : public FileSystemAccess
     bool mkdirlocal(LocalPath&, bool) override;
     bool setmtimelocal(LocalPath&, m_time_t) override;
     bool chdirlocal(LocalPath&) const override;
-    bool getextension(const LocalPath&, char*, size_t) const override;
+    bool getextension(const LocalPath&, string) const override;
```
Comment (polmr) [inline include/mega/win32/megafs.h:81]:
```suggestion
    bool getextension(const LocalPath&, string&) const override;
```

**Review by polmr — CHANGES_REQUESTED** [review]

**Comment by polmr** [comment]
Now the issue for windows is in megacli:
```
megacli.cpp(6414): error C2660: 'mega::FileSystemAccess::getextension': function does not take 3 arguments 
include\mega\filesystem.h(506): note: see declaration of 'mega::FileSystemAccess::getextension'
```
almost there :sweat_smile:

**Pull request closed without merging**